src/garage/tf/algos/rl2.py

```diff
@@ -1,9 +1,13 @@
-"""Wrapper class that is specifc for RL2."""
+"""Module for RL2.
+
+This module contains RL2Worker and the environment wrapper for RL2.
+"""
 import akro
 import gym
 import numpy as np
 
 from garage.envs.env_spec import EnvSpec
+from garage.sampler.worker import DefaultWorker
```

Comment:
is DefaultWorker not exported in `garage.sampler`? Same question for `EnvSpec`

Reply:
Will fix